### PR TITLE
[feature] Add reusable handler to utils, normalize all bot handlers

### PR DESCRIPTION
**Pull request by brendacs**
### Purpose of pull request

**Related Issue**: resolves N/A

Reusable handler to enforce pattern across bots, update bots to use new handler

### Pull request checklist

- [x] Branch and PR are named correctly
- [ ] Updated relevant documentation
- [x] Tested with a tester bot
- [x] Wrote unit tests for changes

### Testing instructions

Salary and AMA bot still work

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
+
+  if (!subCommand) {
+    client.message.channel.send(getCommandsString());
+    return;
+  }
+
+  let targetCmd = subCommands[subCommand];
+
+  if (!targetCmd) {
+    client.message.channel.send(strings.invalidSubCommand);
```
Comment (sirNikolai) [inline src/utils/index.js:21]: one thing I did change was make this one string. Better one with nl than 2 pings of server

Comment (brendacs) [reply src/utils/index.js]: yeah, I had noticed and updated

File: src/utils/index.js [hunk 2]
```diff
@@ -39,4 +45,27 @@ export function getCommandsString(subCommands) {
   str += escapedBackticks;
 
   return str;
-}
+};
+
+/**
+ * Parses message content for commands and arguments
+ * @returns {Object.<string, (string | Array.<string>)>} - an object with the subCommand and arguments
```
Comment (sirNikolai) [inline src/utils/index.js:52]: should subCommand be `sub command` here?

**Review by sirNikolai — APPROVED** [review]
lgtm, need to change pathname for ama which git lovingly butchered.

**Pull request merged**